python/lsst/ts/xml/data/sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -494,4 +494,82 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_electricFieldStrength</EFDB_Topic>
+    <Description>Electric field strength for a period of 10 seconds (20 measurements per second).</Description>
```

Comment:
I suggest deleting  "(20 measurements per second)" because it may not be true for all sensors and it may be configurable for some.

I worry that delaying this data makes it less useful, so I'm glad you have the high field event as well as this telemetry.

I strongly suggest make it 1/second instead of 1/10 seconds so it has more value as a warning. Or make the timespan configurable, in which case change the "for a period of 10 seconds" to something like "over a configurable period".